examples/target-group-policy.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: application-networking.k8s.aws/v1alpha1
+kind: TargetGroupPolicy
+metadata:
+    name: test-policy
+spec:
+    targetRef:
+        group: ""
```

Comment:
If we leave the ```group``` field out does applying the YAML complain or does it simply set group to ""?

Reply:
it will simply set group to empty string, btw empty group means "core" in k8s which is a common convention